### DISCUSSION TITLE
Disputed territories/states with limited recognition

### DISCUSSION THREAD

**Discussion by Forrest-J** [owid/covid-19-data#2086] [Ideas]
How are disputed territories and states with limited recognition which have independent vaccination initiatives counted in the data? Will they be added just like other countries if their figures are not counted in any other country's figures? As far as I have found, Abkhazia, Luhansk PR, Donetsk PR, South Ossetia, Transnistria, Artsakh, and Sahrawi Arab DR have all began their respective vaccination campaigns.

**Comment by Forrest-J** [comment]
Is this issue going to be addressed? If not, Wikipedia can account for these regions separately.

FYI, relevant sources from comments in the main issue: [Abkhazia](https://github.com/owid/covid-19-data/issues/230#issuecomment-850532892), [Luhansk PR](https://github.com/owid/covid-19-data/issues/230#issuecomment-850550925), [Donetsk PR](https://github.com/owid/covid-19-data/issues/230#issuecomment-850571072), [South Ossetia](https://github.com/owid/covid-19-data/issues/230#issuecomment-850578819), [Transnistria](https://github.com/owid/covid-19-data/issues/230#issuecomment-850607292), [Artsakh](https://github.com/owid/covid-19-data/issues/230#issuecomment-850623858), [Sahrawi Arab DR](https://github.com/owid/covid-19-data/issues/230#issuecomment-850641692)

**Comment by lucasrodes** [comment]
Hi @Forrest-J,
Thanks for reporting.

We have noted the sources and will be discussing/reviewing this once we have the time.
